system-test/trace-express.js

```diff
@@ -134,7 +134,7 @@ describe('express + datastore', () => {
       const trace = traces[0];
       if (usingAsyncHooks) {
         assert.equal(trace.spans.length, 3, 'should be 3 spans: parent, child, auth');
-        assert.equal(trace.spans[2].name, 'accounts.google.com');
+        assert.equal(trace.spans[2].name, 'www.googleapis.com');
```

Comment:
Was this failing before?

Reply:
No, it actually didn't fail before, surprisingly. My guess is that it's a strange artifact of npm having picked a different patch-level version of the auth library based on the larger dependency tree that it had to construct when all of the dependencies were listed in a single package.json file.